core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -284,8 +287,27 @@ export class TranslationModalComponent {
     for (const [i, originalElement] of filteredOriginalElements.entries()) {
       if (originalElement.nodeName !== filteredTranslatedElements[
         i].nodeName) {
+        this.elementViolation = (
+          '"' + filteredTranslatedElements[i].outerHTML + '"' + ' does ' +
+          'not match ' + '"' + originalElement.outerHTML + '"' + '.');
```

Comment:
Since the help text is always the same, we can extract the error message into a function that returns an interpolated string that takes in the actual and expected html as parameters.

Reply:
Done